gulpfile.js

```diff
@@ -1,21 +1,21 @@
 'use strict';
 
-var gulp = require('gulp');
-var sass = require('gulp-sass');
+const gulp = require('gulp');
```

Comment:
For all practical purposes it may not make a difference, but this change implies that different versions of node are supported. I checked, and complete const support was added ages ago so it should be fine.